chanzuckerberg/idseq-web
# Add saving of table or taxon from the report page

# Description

This PR builds on #1930 and #1931 to add saving of table or taxon from the report page. (Sorry, #1930 
 changes are repeated here. Once that merges, they will go away.) 

![image](https://user-images.githubusercontent.com/28797/52758358-c4958080-2fbc-11e9-94f6-b34cf5ffe12a.png)

In the next PR, I will add a way to load those saved visualizations. (It's not clear whether they should be loaded on the same multipurpose page or each on a new standalone /visualizations/type/ page.)

## Type of change


# Test and Reproduce
Outline the steps to test or reproduce the PR here.

1. go to sample report page
2. hit save button
3. see "success"
4. make sure heatmaps save and load still work
5. try "share" button

## Impacted Areas in Application
List general components of the application that this PR will affect:

Which pages?

# Checklist:
